### ISSUE TITLE
imgui content doesn't render on iOS

### ISSUE THREAD

**Issue by dri-richard** [britzl/extension-imgui#15]
To reproduce:

1. Bundle the example project for iOS, targeting 64bit
2. Install on device & run
3. Observe that only the sprite is visible, and the imgui demo window is not rendered

**Comment by dri-richard** [comment]
A debug build crashed, with the following output
(Defold 1.2.185, iPhone SE 2, iOS 14.7.1)

```
INFO:DEFOLDIMGUI: imgui_ImageLoad example/images/image1.png
ERROR:DEFOLDIMGUI: Error loading image: example/images/image1.png
INFO:DEFOLDIMGUI: imgui_ImageInternalLoad before 1
INFO:DEFOLDIMGUI: imgui_ImageInternalLoad after 4
INFO:CRASH: Successfully wrote Crashdump to file: /var/mobile/Containers/Data/Application/DAD78D89-0E09-4FCF-BB0F-F2703220E063/Library/Application Support/Defold/_crash
ERROR:CRASH: CALL STACK:
ERROR:CRASH: 0   ImGui                               0x000000010459d8b0 _ZN7dmCrash7OnCrashEi + 104
ERROR:CRASH: 1   libsystem_platform.dylib            0x00000001d900029c 987D48D5-3CAA-34D7-9141-C0BD6E2351E8 + 21148
ERROR:CRASH: 2   libsystem_pthread.dylib             0x00000001d9005a9c pthread_kill + 272
ERROR:CRASH: 3   libsystem_c.dylib                   0x00000001966e0b84 abort + 124
ERROR:CRASH: 4   libsystem_c.dylib                   0x00000001966e0004 err + 0
ERROR:CRASH: 5   ImGui                               0x000000010444543c _ZN10dmGraphicsL24OpenGLDeleteVertexBufferEm + 0
ERROR:CRASH: 6   ImGui                               0x0000000104427f08 _ZN12dmGameSystem16CompSpriteRenderERKN12dmGameObject22ComponentsRenderParamsE + 992
ERROR:CRASH: 7   ImGui                               0x00000001043ec814 _ZN12dmGameObject6RenderEPNS_16CollectionHandleE + 260
ERROR:CRASH: 8   ImGui                               0x00000001043aa8d8 _ZN8dmEngine4StepEPNS_6EngineE + 820
ERROR:CRASH: 9   ImGui                               0x00000001043ab8e8 _Z14dmEngineUpdatePN8dmEngine6EngineE + 32
ERROR:CRASH: 10  ImGui                               0x000000010457f1bc -[AppDelegate appUpdate] + 36
ERROR:CRASH: 11  QuartzCore                          0x000000019057f7b8 03F2F6C8-9BEC-3525-81AA-6125D428FBB7 + 83896
ERROR:CRASH: 12  QuartzCore                          0x000000019065e7f4 03F2F6C8-9BEC-3525-81AA-6125D428FBB7 + 997364
ERROR:CRASH: 13  CoreFoundation                      0x000000018d26ea0c 338F58B1-9B75-38A0-B908-E2574AB1D39A + 518668
ERROR:CRASH: 14  CoreFoundation                      0x000000018d29421c 338F58B1-9B75-38A0-B908-E2574AB1D39A + 672284
ERROR:CRASH: 15  CoreFoundation                      0x000000018d29359c 338F58B1-9B75-38A0-B908-E2574AB1D39A + 669084
ERROR:CRASH: 16  CoreFoundation                      0x000000018d28d180 338F58B1-9B75-38A0-B908-E2574AB1D39A + 643456
ERROR:CRASH: 17  CoreFoundation                      0x000000018d28c308 CFRunLoopRunSpecific + 600
ERROR:CRASH: 18  GraphicsServices                    0x00000001a490f734 GSEventRunModal + 164
ERROR:CRASH: 19  UIKitCore                           0x000000018fd0a75c 41D9BF62-8439-3B21-9959-4E27E75C294E + 12363612
ERROR:CRASH: 20  UIKitCore                           0x000000018fd0ffcc UIApplicationMain + 168
ERROR:CRASH: 21  ImGui                               0x00000001043abaec _ZN8dmEngine7RunLoopEPKNS_13RunLoopParamsE + 44
ERROR:CRASH: 22  ImGui                               0x00000001043abaac _Z11engine_mainiPPc + 96
ERROR:CRASH: 23  libdyld.dylib                       0x000000018cf48cf8 FF7D2567-9277-3AA6-B3DD-2C5748E432A6 + 7416
```

**Comment by britzl** [comment]
I believe this has been fixed in commit https://github.com/britzl/extension-imgui/commit/454025a0babb9e4e7a672a05a12886dd25ce6675

**Comment by dri-richard** [comment]
Verified fixed in 193fe5d but has compilation errors on Windows:

/imgui/src/extension_imgui.cpp
Line 89: use of undeclared identifier ‘GL_CLAMP_TO_EDGE’
Line 90: use of undeclared identifier ‘GL_CLAMP_TO_EDGE’
Line 1663: use of undeclared identifier ‘gl3wInit’; did you mean ‘glfwInit’?
Line 1664: use of undeclared identifier ‘GL3W_OK’

**Comment by britzl** [comment]
> Verified fixed 

Thanks!

> Has compilation errors on Windows

Created: https://github.com/britzl/extension-imgui/issues/17
